config/routes.rb

```diff
@@ -9,4 +9,14 @@
       resources :likes, only: [:create]
     end
   end
+
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [:index, :show, :create] do
+        resources :posts, only: [:index, :show, :create] do
+          resources :comments, only: [:index, :show, :create]
+        end
+      end
+    end
+  end
```

Comment:
- [ ] Kindly fix this issue. You have the right directory but you don't have users_controller file in your api/v1 folder. 
![image](https://user-images.githubusercontent.com/91486791/233799548-dd71b8fe-fd03-46de-84f6-9580c4dd1a0d.png)

Reply:
Thanks @tugsem 
We've removed the routes we didn't need on the routes.rb file.